sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -402,13 +402,9 @@ private static FormField setFormField(String name, FieldData valueData, FieldVal
                 value = new com.azure.ai.formrecognizer.models.FieldValue(selectionMarkState,
                     FieldValueType.SELECTION_MARK_STATE);
                 break;
-            case GENDER:
-                value = new com.azure.ai.formrecognizer.models.FieldValue(fieldValue.getValueGender(),
-                    FieldValueType.GENDER);
-                break;
-            case COUNTRY:
-                value = new com.azure.ai.formrecognizer.models.FieldValue(fieldValue.getValueCountry(),
-                    FieldValueType.COUNTRY);
+            case COUNTRY_REGION:
```

Comment:
Not specifically this line, but above where we have the workaround to use the `text` value for the selection mark - you should be able to use the valueSelectionMark now that the service fixed the bug in v2.1